specification/ai/data-plane/Face/preview/v1.1-preview.1/Face.json

```diff
@@ -0,0 +1,7948 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure AI Face API",
+    "version": "v1.1-preview.1",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "name": "endpoint",
+        "in": "path",
+        "description": "Supported Cognitive Services endpoints (protocol and hostname, for example:\nhttps://<resource-name>.cognitiveservices.azure.com).",
+        "required": true,
+        "type": "string"
+      }
+    ]
+  },
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "ApiKeyAuth": []
+    },
+    {
+      "OAuth2Auth": [
+        "https://cognitiveservices.azure.com/.default"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "ApiKeyAuth": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    },
+    "OAuth2Auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "https://cognitiveservices.azure.com/.default": ""
+      },
+      "tokenUrl": "https://login.microsoftonline.com/common/oauth2/token"
+    }
+  },
+  "tags": [],
+  "paths": {
+    "/face/{apiVersion}/compare": {
```

Comment:
I think you should move "/face/{apiVersion}" into the `x-ms-parameterized-host`, as it was in your GA API. That will be more DRY and also make comparison across API versions easier.

Reply:
Updated. However, `Azure.Core.Foundations.ApiVersionParameter` is injected to all our API's parameter, I could not find a way to remove it.

Reply:
I'm going to rollback the change since there is no way to remove ApiVersionParameter in query if we don't override it by putting it in request path.